### ISSUE TITLE
Swaylock doesn't handle faillock properly.

### ISSUE THREAD

**Issue by notmentaloutlaw** [swaywm/swaylock#286]
If my pam is configured so if I mess up 3 times it locks my account for 10 mins. So If I login fail twice but then succeed the third time it resets my 'attempts'. With swaylock if you fail twice succeed the third but then lock your screen again and fail one more time it will count as you failed 3 times in a row and lock your account. This is because the entries are still seen in faillock. It does not reset entries when someone successfully logs in. 

```
# faillock before starting swaylock
When                Type  Source                                           Valid

# faillock after failing two times and success on third
When                Type  Source                                           Valid
2023-02-09 21:13:31 SVC   swaylock                                             V
2023-02-09 21:13:34 SVC   swaylock                                             V

# This should not look like this this should look like this
When                Type  Source                                           Valid
```

If you do this for system logins on tty you will see that it does this correctly and does not have this problem. For now users can just do faillock --reset after they logged in or add that to your keybind. But this shouldn't be needed if its configured properly. I am not that familiar with pam but maybe swaylock doesn't send the message to pam after its unlocked that it is a successful login attempt?

**Comment by BetaRays** [comment]
Another issue with faillock is that swaylock will not print anything when the account is locked, though a message is supposed to be sent with `pam_info` by faillock. I think this is what was asked in https://github.com/swaywm/swaylock/issues/213 too.
